lib/ansible/utils/module_docs_fragments/mysql.py

```diff
@@ -47,6 +47,11 @@ class ModuleDocFragment(object):
       - The path to a Unix domain socket for local connections
     required: false
     default: null
+  connect_timeout:
```

Comment:
@bcoca: Good catch. Sorry about that. I've added the missing version_added entry.